test/unit-tests/components/views/settings/SetIdServer-test.tsx

```diff
@@ -56,13 +56,15 @@ describe("<SetIdServer />", () => {
         const identServerField = getByLabelText("Enter a new identity server");
         await userEvent.type(identServerField, "https://identity.example.org");
         await userEvent.click(getByRole("button", { name: "Change" }));
+        await waitFor(() => expect(getByRole("button", { name: "Continue" })).toBeInTheDocument());
```

Comment:
why not use `await findByRole` either here or in the line below, that waits just like `waitFor`

Reply:
Why else would it be async?

Reply:
Good point